manual/manual/library/libunix.etex

```diff
@@ -57,39 +57,41 @@ the Unix module.  The native Win32 ports implement a subset of them.
 Below is a list of the functions that are not implemented, or only
 partially implemented, by the Win32 ports. Functions not mentioned are
 fully implemented and behave as described previously in this chapter.
+\end{windows}
 
 \begin{tableau}{|l|p{8cm}|}{Functions}{Comment}
 \entree{"fork"}{not implemented, use "create_process" or threads}
 \entree{"wait"}{not implemented, use "waitpid"}
 \entree{"waitpid"}{can only wait for a given PID, not any child process}
 \entree{"getppid"}{not implemented (meaningless under Windows)}
 \entree{"nice"}{not implemented}
-\entree{"truncate", "ftruncate"}{not implemented}
+\entree{"truncate", "ftruncate"}{implemented (since 4.10.0)}
 \entree{"link"}{implemented (since 3.02)}
-\entree{"symlink", "readlink"}{implemented (since 4.03.0)}
-\entree{"access"}{execute permission "X_OK" cannot be tested,
-  it just tests for read permission instead}
 \entree{"fchmod"}{not implemented}
 \entree{"chown", "fchown"}{not implemented (make no sense on a DOS
 file system)}
 \entree{"umask"}{not implemented}
+\entree{"access"}{execute permission "X_OK" cannot be tested,
+  it just tests for read permission instead}
+\entree{"chroot"}{not implemented}
 \entree{"mkfifo"}{not implemented}
+\entree{"symlink", "readlink"}{implemented (since 4.03.0)}
 \entree{"kill"}{partially implemented (since 4.00.0): only the "sigkill" signal
 is implemented}
+\entree{"sigprocmask", "sigpending", "sigsuspend"}{not implemented (no inter-process signals on Windows}
 \entree{"pause"}{not implemented (no inter-process signals in Windows)}
 \entree{"alarm"}{not implemented}
 \entree{"times"}{partially implemented, will not report timings for child
 processes}
 \entree{"getitimer", "setitimer"}{not implemented}
-\entree{"getuid", "geteuid", "getgid", "getegid"}{always return 1}
+\entree{"getuid", "geteuid", "getgid", "getegid"}{always returns 1}
+\entree{"setuid", "setgid", "setgroups", "initgroups"}{not implemented}
 \entree{"getgroups"}{always returns "[|1|]" (since 2.00)}
-\entree{"setuid", "setgid", "setgroups"}{not implemented}
-\entree{"getpwnam", "getpwuid"}{always raise "Not_found"}
-\entree{"getgrnam", "getgrgid"}{always raise "Not_found"}
+\entree{"getpwnam", "getpwuid"}{always raises "Not_found"}
+\entree{"getgrnam", "getgrgid"}{always raises "Not_found"}
```

Comment:
Similarly, given that they're formatted as a list?

Reply:
Done.